ckanext/stadtzhtheme/fanstatic/autocomplete-ogdzh_search.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+ckan.module('autocomplete-ogdzh_search', function ($) {
+  return {
+    initialize: function () {
+        new autoComplete({
+            selector: 'input#ogdzh_search[name="q"]',
+            minChars: 2,
+            renderItem: function (item, search){
+                // put some searches in quotes, so that solr does not interpret special characters
+                var pattern = /\s/;
+                if (search.match(pattern)) {
+                    item = '"' + item + '"';
+                }
+                return '<div class="autocomplete-suggestion" data-val="' + item.replace(/"/g, '&quot;') + '">' + item + '</div>';
+            },
+            source: function(term, response){
+                var url = '/api/3/action/ogdzh_autosuggest';
+                var params = {q: term};
+                // check if any filters/facets are set and send them along
+                var values = [];
```

Comment:
The readme needs to be updated as well as it points to the file that no longer exists.